great_expectations/experimental/metric_repository/metric_list_metric_retriever.py

```diff
@@ -44,8 +48,14 @@ def get_metrics(
         )
         metrics_result.extend(table_metrics)
 
-        # exit early if only Table Metrics exist
         if not self._column_metrics_in_metric_list(metric_list):
+            # if no column metrics are present in the metric list, we can return the table metrics
+            return metrics_result
+
+        if MetricTypes.TABLE_COLUMN_TYPES not in metric_list:
+            logger.warning(
+                "TABLE_COLUMN_TYPES metric is required to compute column metrics. Skipping column metrics."
```

Comment:
column metrics cannot be returned unless `TABLE_COLUMN_TYPES` is present

Reply:
Cool, thanks. I think that requiring the TABLE_COLUMN_TYPES metric for now makes sense and is prob the easiest way to go. In the future, if we have to support fetching metrics without returning the column types, maybe there is a way we can fetch the column types and use them to calculate the other metrics and just not return the types.

(Just thinking of a use case where we may need to split up compute jobs to have a smoother/more progressive loading experience once we add more graphs to the Data Asset Overview page.)